host/app/components/schema-inspector.gts

```diff
@@ -72,12 +72,37 @@ export default class SchemaInspector extends Component<{ Args: { module: Record<
   }
 
   @action
-  save() {
+  async save() {
     if (!this.newInstance || !this.selected) {
       return;
     }
     // TODO: pick filename and write JSON file
     console.log(this.newInstance, moduleURL(this.args.module), this.selected.name);
+
+    let json = serializeCard(this.newInstance);
```

Comment:
we need to add the meta to the JSON which includes the module URL and export name